### ISSUE TITLE
0.9.4

### ISSUE THREAD

**Issue by skoczen** [skoczen/will#254]
Hey everyone,

**TL;DR**: Will lives! `pip install -e git://github.com/skoczen/will.git@master#egg=will`, report any bugs, and **thank you**!!

Just a quick update on the next Will release - I've merged in most open PRs into master, and it appears stable here. I'm aiming for a release in the coming week.
 
**However**, since folks have been running various mission-critical plugins for the past year plus, and things got a bit stale, I could use a few more eyes on this release, to make sure it's up to the usual standards.

So, in that vein, tagging a few frequent contributors here for some testing, (please tag anyone else you think would be great!)  Cheers, @woohgit @brandonsturgeon @borgstrom @netjunkie @poirier @camilonova @BrianGallew @rbp 

Once we get this out, bigger news and conversations around will will get properly started.  Can't wait to talk more about the direction, listen to thoughts, and get moving!

Thanks so much, everyone!!

-Steven

**Comment by woohgit** [comment]
I've just rolled out the master branch to production. Actively monitoring it ;)

**Comment by skoczen** [comment]
@woohgit Thanks for rolling it out!  How's it looking?

**Comment by skoczen** [comment]
Since will is running well here and I haven't heard any reports, releasing 0.9.4 this afternoon - thanks to all for your contributions, and look forward to digging into the future of will! :)
